src/main.tsx

```diff
@@ -1,10 +1,15 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import ReactDOM from 'react-dom/client';
+import App from './App.tsx';
+import './index.css';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { render } from 'react-dom';
+const queryClient = new QueryClient();
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
+  // <React.StrictMode>
+  <QueryClientProvider client={queryClient}>
     <App />
-  </React.StrictMode>,
-)
+  </QueryClientProvider>,
+
+  // </React.StrictMode>,
```

Comment:
Remove if not used please.

Reply:
I toggle these, so I do use them. I'll add a comment